### PR TITLE
[Vue] migrate import version directive to vue component

### PR DESCRIPTION
**Pull request by diosmosis**
### Description:

Changes:
* migrate import version directive
* wait for .modal.open before taking screenshot

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: vue/src/ImportVersion/ImportVersion.adapter.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+export default createAngularJsAdapter({
+  component: ImportVersion,
+  scope: {
+    idContainer: {
+      angularJsBind: '=',
+    },
+    onSomething: {
+      angularJsBind: '&',
+    },
```
Comment (sgiehl) [inline vue/src/ImportVersion/ImportVersion.adapter.ts]: What is that one used for?

File: vue/src/ImportVersion/ImportVersion.vue [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+          },
+        );
+      } else {
+        this.showNotification(translate('TagManager_ErrorContainerVersionImportIncomplete'), 'error');
+      }
+    },
+  },
+  computed: {
+    backupNameTitle() {
+      return `${translate('TagManager_BackupVersionName')} (${translate('General_Recommended')})'`;
```
Comment (sgiehl) [inline vue/src/ImportVersion/ImportVersion.vue]:
```suggestion
      return `${translate('TagManager_BackupVersionName')} (${translate('General_Recommended')})`;
```

**Review by sgiehl — CHANGES_REQUESTED** [review]
Left some minor comments. Otherwise looks good to merge

File: vue/src/ImportVersion/ImportVersion.adapter.ts (hunk 1 above)
Comment (diosmosis) [reply vue/src/ImportVersion/ImportVersion.adapter.ts]: Nothing, this was part of a test, will remove it.

Comment (sgiehl) [reply vue/src/ImportVersion/ImportVersion.adapter.ts]:
```suggestion
```

**Review by sgiehl — APPROVED** [review]

**Pull request merged**